erlang/src/erl_parse.ml

```diff
@@ -10,3 +11,11 @@ let from_file source_file =
       in
       Error (`Parser_error msg)
   | x -> Ok x
+
+let exprs_from_file source_file = 
```

Comment:
I think the intention here is to parse Erlang _terms_ so lets rename this to be `terms_from_file.

While we're at it, we can also make the other function be `module_from_file` to be a bit more explicit

Reply:
done. except make fmt as I'm on dune 2.6 and not able to compile caramel.
sorry!
I remembered to specialise tuple() for exprs only to tuple_for_expr()
(as for expr_list) to capture comments.